### DISCUSSION TITLE
ARM Ripping of Audio Description audio track from DVD/BlueRay

### DISCUSSION THREAD

**Discussion by sdjernes** [automatic-ripping-machine/automatic-ripping-machine#800] [Q&A]
What setting would I change to make sure along with the standard surround track and Sterio track the Audio Description track is ripped and included in the resultant MKV file?

**Comment by shitwolfymakes** [comment]
That is part of the MakeMKV features, so you'd have to read their documentation on that.

Once you know what arguments to pass, you need to go to the ripper settings page and add them to the `MKV_ARGS`field, and save the updated settings. The next disk you rip will then use those arguments
